calico_containers/calicoctl.py

```diff
@@ -388,6 +374,70 @@ def node(ip, node_image, ip6=""):
     print "Calico node is running with id: %s" % cid
 
 
+def checksystem(fix=False, quit_if_error=False):
+    """
+    Checks that the system is setup correctly. If --fix is passed in at command line, this command
+    will attempt to fix any issues it encounters. If any fixes fail, it will exit(1)
+
+    :return: True if all system dependencies are in the proper state, False if they are not, and exit with status 1
+             if the fix flag was passed in and they could not be fixed
+    """
+    if fix:
+        # modprobe and sysctl require root privileges to make modifications.
+        enforce_root()
+
+    modprobe = sh.Command._create('modprobe')
+    system_ok = True
+    if not module_loaded("ip6_tables"):
+        if fix:
+            try:
+                modprobe('ip6_tables')
+            except:
+                print >> sys.stderr, "ERROR: calico could not enable ip6_tables."
```

Comment:
I think we should drop "calico" here.
Just write
ERROR: Could no enable ip6_tables
and
WARNING: Unable to detect the ip6_tables module. Load with...